README.md

```diff
@@ -388,3 +388,19 @@ specifically disclaims any liability with respect to this document
 and no contractual obligations are formed either directly or
 indirectly by this document. The technologies, functionality, services,
 and processes described herein are subject to change without notice.
+
+## **Important Notice: This Repository is Deprecated**
+
+**Important Notice**
+
+This repository (fsxa-pattern-library) has been deprecated and is no longer actively maintained. Its usage comes with potential risks and limitations. We recommend users to consider alternative solutions for their needs.
+
+**Alternative Solution:**
+
+The functionality and features of this repository have been transitioned to the `content-api` repository, which is the new replacement for FSXA stack. You can find the latest updates, improvements, and ongoing support there.
```

Comment:
Correct me if I am wrong or blind, but I see no link here? Is it somehow magically hidden and will show up when opened in the regular browser html view? How is the customer supposed to know which repository you mean?